app/src/main/java/org/dhis2/usescases/searchTrackEntity/adapters/BaseViewHolder.kt

```diff
@@ -0,0 +1,107 @@
+package org.dhis2.usescases.searchTrackEntity.adapters
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.dhis2.Bindings.addEnrollmentIcons
+import org.dhis2.Bindings.hasFollowUp
+import org.dhis2.Bindings.setAttributeList
+import org.dhis2.Bindings.setStatusText
+import org.dhis2.Bindings.setTeiImage
+import org.dhis2.Bindings.toDateSpan
+import org.dhis2.databinding.ItemSearchTrackedEntityBinding
+import org.dhis2.usescases.searchTrackEntity.SearchTEContractsModule
+
+abstract class BaseViewHolder(
```

Comment:
Reading baseViewHolder it seems that it might be something for using in any part of the app, but you can't since it's couple to the teiModel. May be a more accurate name could be BaseTeiViewHolder?